examples/playground/src/app.js

```diff
@@ -3,6 +3,7 @@ import {render} from 'react-dom';
 import AutoSizer from 'react-virtualized-auto-sizer';
 import {StaticMap} from 'react-map-gl';
 import DeckWithMaps from './deck-with-maps';
```

Comment:
Can we type things out? `DeckWithMapboxMaps` and `DeckWithGoogleMaps` from `deck-with-google-maps`?